pandas/tests/extension/base/getitem.py

```diff
@@ -324,11 +324,11 @@ def test_take_non_na_fill_value(self, data_missing):
         fill_value = data_missing[1]  # valid
         na = data_missing[0]
 
-        array = data_missing._from_sequence(
+        d_arr = data_missing._from_sequence(
```

Comment:
as suggested in https://github.com/pandas-dev/pandas/pull/40286#discussion_r589747955 , can't this just be `arr`? It's not clear to me what `d_arr` is mean to stand for